plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -142,8 +146,6 @@ public StyledString getStyledText(Object element) {
 			case ResourceKind.SERVICE:
 				IService service = (IService) resource;
 				return getStyledText(service);
-			case ResourceKind.TEMPLATE:
```

Comment:
This class was updated to be a common label provider anywhere in the UI where we need such capability.  I think it would be reasonable to leave the others there.  We may need to add template back in at some time if we ever wish to provide a tab or some other visibility into templates other then the way we display them in the list page.